app/types/date_list_type.rb

```diff
@@ -0,0 +1,7 @@
+class DateListType < ActiveModel::Type::Value
+  def cast(value)
```

Comment:
Should we keep thinkgs consistant and call `ContactsEnumerableType`  => `ContactsListType`

`s/enumberable/list`

Reply:
`ContactEnumerable` (as a typed list) couldn't name it `ContactList` because there is already a class called `ContactList` and the type cast classes lookup the type based on their name by default so had to call them `ContactEnumerable` and `ContactEnumerableType`.

There would be a lot less boiler plate code and easier naming if attributes supported supported arrays like it does for Postgres.

i.e.

```ruby
attribute :prices, :integer, array: true
```